Documentation/helm-operator.md

```diff
@@ -123,20 +124,25 @@ The following tables lists the configurable parameters of the rook-operator char
 | `csi.cephFSProvisionerNodeAffinity` | The node labels for affinity of the CephCSI CephFS provisioner deployment (***)                                             | <none>                                                    |
 | `csi.cephFSPluginTolerations`       | Array of tolerations in YAML format which will be added to CephCSI CephFS plugin DaemonSet                                  | <none>                                                    |
 | `csi.cephFSPluginNodeAffinity`      | The node labels for affinity of the CephCSI CephFS plugin DaemonSet (***)                                                   | <none>                                                    |
+| `csi.cephNFSPluginTolerations`       | Array of tolerations in YAML format which will be added to CephCSI CephNFS plugin DaemonSet                                  | <none>                                                    |
```

Comment:
should we call it as   `CephCSI NFS` instead of `CephCSI CephNFS`

Reply:
I vote yes.

Reply:
`Ceph-CSI NFS` would be most correct, but `CephCSI` is already used, so stick with that?

Reply:
done